### PR TITLE
Policy management paper

### PR DESCRIPTION
**Pull request by JimBugwadia**
This PR is the commit of the `Kubernetes Policy Management` paper from WG-Policy.

The paper was drafted and reviewed at:  https://docs.google.com/document/d/1jz384UwLSMN4fEbHv0oBao-J7YquYhKx18w_jwKtX9g/edit#.

This is to transfer the paper from Google docs to Git. 

Initially the paper was housed at https://github.com/kubernetes-sigs/wg-policy-prototypes but since working groups are temporary, a more permanent home was discussed in the SIG Security docs meeting on Oct 7th, 2021:

https://docs.google.com/document/d/11LZn7qWB0OzbpF8va_YYGQE4fuRARCGY9KL87hwBLBI/edit#

```
[Robert] Policy WG would like to see if docs repo is the correct (or acceptable) landing place for our policy whitepaper?
[Robert] Looking for a home to add the whitepaper
[Savitha] k/sig-security sounds like a good place +1[Rory]
```

### CODE REVIEW

**Comment by JimBugwadia** [comment]
- [markdown preview](https://github.com/kubernetes/community/blob/3a7c1f467c4fc6f12de054414aaf87acc5abaa46/sig-security/policy/kubernetes-policy-management.md)
- [PDF preview](https://github.com/kubernetes/community/blob/3a7c1f467c4fc6f12de054414aaf87acc5abaa46/sig-security/policy/kubernetes-policy-management.pdf)

**Comment by PushkarJ** [comment]
@JimBugwadia Just a heads up, we are in the process of moving some content to https://github.com/kubernetes/sig-security that is under `k/community/sig-security`. This paper would be a good candidate to move under the new repo too.

If you'd like to wait until the move happens and the repo is ready to consume the paper, feel free to put this on hold. If not, no worries, we will move it after this is merged :) cc @savitharaghunathan

**Comment by sftim** [comment]
>  process to add CNCF branding, etc

I'm not sure on that. I would try asking the CNCF directly @JimBugwadia.

**Comment by JimBugwadia** [comment]
> @JimBugwadia Just a heads up, we are in the process of moving some content to https://github.com/kubernetes/sig-security that is under k/community/sig-security. This paper would be a good candidate to move under the new repo too.
>
> If you'd like to wait until the move happens and the repo is ready to consume the paper, feel free to put this on hold. If not, no worries, we will move it after this is merged :) cc @savitharaghunathan

@PushkarJ @savitharaghunathan - I am open to your suggestions on what works best here.

**Comment by PushkarJ** [comment]
After discussing this with @rficcaglia in today's meeting, we will not merge this PR. Instead once https://github.com/kubernetes/sig-security/issues/9 is merged, a new PR would need to be created to put the paper under k/sig-security/docs directory. Please add a comment here for any questions or concerns!!

/hold

File: sig-security/policy/kubernetes-policy-management.md [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+* Alok Raj
+* Dan Papandrea, Sysdig
+* Abdelrahman Essam
+* Rahul Jadhav, Accuknox
+
+The authors also wish to thank the chairs and members of the CNCF [TAG Security](https://github.com/cncf/tag-security#readme) and [Kubernetes SIG Security](https://github.com/kubernetes/community/tree/master/sig-security#security-special-interest-group) groups for their collaboration and support.
+
+## Target Audience
+
+This paper is intended for Kubernetes administrators, Site Reliability Engineers (SREs), and other practitioners wanting to Kubernetes clusters and workloads in compliance to organizational standards as well as regulatory compliance standards for security, resiliency, and best practices.
```
Comment (TheLunaticScripter) [inline sig-security/policy/kubernetes-policy-management.md:84]:
```suggestion
This paper is intended for Kubernetes administrators, Site Reliability Engineers (SREs), and other practitioners wanting to create Kubernetes clusters and workloads in compliance with organizational standards as well as regulatory compliance standards for security, resiliency, and best practices. 
```

**Comment by JimBugwadia** [comment]
Hi @celestehorgan, thanks for the great comments and feedback! I've incorporated changes accordingly. Can you please do a quick check?

**Comment by JimBugwadia** [comment]
Hi @TheLunaticScripter - thanks for the review! I've made the requested changes.

**Comment by JimBugwadia** [comment]
previews:
- [markdown](https://github.com/kubernetes/community/blob/c61508a8651fcb49036188410becc36a3750217b/sig-security/policy/kubernetes-policy-management.md)
- [PDF](https://github.com/kubernetes/community/blob/c61508a8651fcb49036188410becc36a3750217b/sig-security/policy/kubernetes-policy-management.pdf)

**Comment by PushkarJ** [comment]
@JimBugwadia The great refactor of https://github.com/kubernetes/sig-security/issues/9 is now complete 🎉 . So we are ready to move this paper to `k/sig-security` repo under `sig-security-docs`

Also noticed that @rficcaglia has a separate PR: https://github.com/kubernetes/sig-security/pull/15 for the same paper. Are the contents in this PR and the one from Robert identical?

**Comment by JimBugwadia** [comment]
@PushkarJ awesome! Thanks for the follow-up!

No, the latest contents (from @celestehorgan's review) are at: https://github.com/JimBugwadia/community/tree/policy-management-paper/sig-security/policy.

I can create a new PR, unless there is a better way to handle.

**Comment by PushkarJ** [comment]
Yes new PR on `k/sig-security` with latest content works. Once that is merged we can close this and the other PR from @rficcaglia

**Comment by JimBugwadia** [comment]
@PushkarJ - created https://github.com/kubernetes/sig-security/pull/17.

**Comment by JimBugwadia** [comment]
Closing this PR. Moved to https://github.com/kubernetes/sig-security/pull/17.

**Pull request closed without merging**